Python >= 3.14 warns about 'asyncio.AbstractEventLoopPolicy' deprecation and so
we cannot turn warnings to errors.

--- aiosignal-1.4.0/tox.ini.orig
+++ aiosignal-1.4.0/tox.ini
@@ -25,7 +25,6 @@
 commands =
   {envpython} \
     {[python-cli-options]byte-errors} \
-    {[python-cli-options]warnings-to-errors} \
     -m pytest \
       {tty:--color=yes} \
       {posargs}
